py/picca/fitter2/pk.py

```diff
@@ -52,6 +52,32 @@ def pk_hcd(k, pk_lin, tracer1, tracer2, **kwargs):
 
     return pk
 
+def pk_hcd_Rogers2018(k, pk_lin, tracer1, tracer2, **kwargs):
+    # alternative to pk_hcd with a physically motivated model (Rogers++ 2018)
```

Comment:
Please add docstrings like in https://github.com/igmhub/picca/blob/5c44d4ecfcae7c25e79b7e5ed84f6172e2cd287a/py/picca/utils.py#L141.
Add the equation number, the page and the arxiv id of the paper.

Reply:
None of the functions in this file have these docstrings...

Reply:
I added a docstring with what I know (I don't know the input/output format of pk_hcd).